pkg/server/endpoints/middleware.go

```diff
@@ -254,6 +254,5 @@ func streamInterceptorMux(oldInterceptor, newInterceptor grpc.StreamServerInterc
 }
 
 func isOldAPI(fullMethod string) bool {
-	return strings.HasPrefix(fullMethod, "/spire.api.node.") ||
-		strings.HasPrefix(fullMethod, "/spire.api.registration.")
```

Comment:
Now that only the registration API is identified as an old API, we may simplify the logic to log deprecation warnings. https://github.com/spiffe/spire/blob/master/pkg/server/endpoints/deprecation.go#L33
Also, seems appropriate to update the test TestWrapWithDeprecationLogging that includes a case for "/spire.api.node.Node/FetchBundle" to avoid confusions in the code.

Reply:
Nice catch! Very good. I'll put up a commit.